siliconcompiler/core.py

```diff
@@ -328,8 +328,10 @@ def create_cmdline(self, progname, description=None, switchlist=[]):
                 sys.exit()
             if 'arg_step' in cmdargs.keys():
                 self.set('arg', 'step', cmdargs['arg_step'], clobber=True)
+            if 'fpga_partname' in cmdargs.keys():
```

Comment:
Are these parameters set inside of the `if 'target' in cmdargs.keys():` block because they could be overwritten by the `load_target` call if they aren't already set when it is called?